### DISCUSSION TITLE
Win10 taskbar enhancement?

### DISCUSSION THREAD

**Discussion by KnIfER** [ramensoftware/windhawk#189] [Q&A]
![image](https://github.com/ramensoftware/windhawk/assets/11593903/4426361c-9b4f-440e-8233-dcb9115429b7)

Is it possible to :
1. change the highlight color of current task.
2. use mouse wheel to scroll up and down when there are more that 2 rows of task buttons.
3. when task button is small, the label containing task name is hidden. Is it possible to display The first character on top of the task icon.

**Comment by m417z** [comment]
1. Currently there's no mod for that.
2. No mod for this, too, but [7+ Taskbar Tweaker](https://tweaker.ramensoftware.com/) has an advanced option which adds this functionality.
3. Also, no mod for that at this time.

**Comment by Anixx** [comment]
I think, the underline color for current task is the accent color.

**Comment by PeaceByJesus** [comment]
For #3, that has to do with the icon size, and in which case I find what helps is to "always group according to application ID" (not just when full)  which neither W/11 nor Windhawk enables AFAIK, but  with Explorer Patcher and  [7+ Taskbar Tweaker](https://tweaker.ramensoftware.com/) does,  (at least up to W/11 22H2).  

 For multiple portable Firefox  installs I found that I had to go into about:config and change taskbar.grouping.useprofile, and hit + at the end  and toggle option to True. 

 [2-lvl-TB.jpg](https://postimg.cc/N518GVg8)

However, I found that   Explorer Patcher 22621.4317.67  broke   the Taskbar grouping, meaning all my LibreOffice docs had their own icon, versus a pop up to choose from, leaving me with 24 icons on my multilevel TB (which EP enables, thank God for Val). 

I needed to revert to [3880.66.6 ](https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3880.66.6_92fce8c) and that fixed it. I tried Windhawk before I did, but the only option I found it offers re. this issue  is to ungroup all.

Perhaps few persons have a lot of programs and documents open, and thus there is no option to group all icons according to application even when the TB is not full.
